sdcm/cluster.py

```diff
@@ -3138,16 +3143,16 @@ def node_setup(node):
             if self._experimental():
                 self._node_setup(node=node)
             node.wait_db_up(verbose=verbose)
-            node.remoter.run('sudo yum install -y scylla-gdb',
-                             verbose=verbose, ignore_status=True)
+            # node.remoter.run('sudo yum install -y scylla-gdb',
+            #                  verbose=verbose, ignore_status=True)
             queue.put(node)
```

Comment:
@larisau , we do not want to remove this from all of our testing machines.
Maybe for this test it's ok, but not for others.